addons/point_of_sale/static/src/js/models.js

```diff
@@ -1828,8 +1830,27 @@ exports.Order = Backbone.Model.extend({
         if(options.quantity !== undefined){
             line.set_quantity(options.quantity);
         }
-        if(options.price !== undefined){
-            line.set_unit_price(options.price);
+
+        if(options.price !== undefined || attr.order.fiscal_position){
```

Comment:
Why changing this condition? You no longer call `set_unit_price` when no fiscal position is configured?